src/main/kotlin/com/kotlin/boardproject/domain/file/FileController.kt

```diff
@@ -1,8 +1,8 @@
-package com.kotlin.boardproject.controller
+package com.kotlin.boardproject.domain.file
 
-import com.kotlin.boardproject.auth.LoginUser
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.service.FileService
+import com.kotlin.boardproject.global.annotation.LoginUser
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end